iot-hub/Samples/service/ImportExportDevicesSample/ImportExportDevicesSample.csproj

```diff
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp2.2</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Azure.Devices" Version="1.27.0-preview-*" />
```

Comment:
Should be Version="1.*"

Reply:
The others aren't 1.*…

Reply:
In fact, CleanUpDevicesSample reference version 1.17 of that assembly, which is old. The last 3 are ok. I'll fix mine.